src/components/App/ActiveEpisodeModal/ActiveEpisodeModal.module.scss

```diff
@@ -22,7 +22,7 @@
       margin-bottom: 2rem;
       border-radius: 0.5rem;
       border: 0.25rem solid #fff;
-      box-shadow: 1px 1px 4px 0px rgba(0,0,0,0.2);
+      box-shadow: 0.125rem 0.125rem 0.25rem 0 rgba(0,0,0,0.2);
```

Comment:
1px is 0.0625rem

Reply:
Right, even ma calculator sucks on maths :laughing: Correcting. :)